examples/rust/tracecon/src/main.rs

```diff
@@ -0,0 +1,121 @@
+use anyhow::{anyhow, bail, Result};
+use core::time::Duration;
+use libbpf_rs::PerfBufferBuilder;
+use object::Object;
+use object::ObjectSymbol;
+use std::convert::TryInto;
+use std::fs;
+use std::net::Ipv4Addr;
+use std::path::Path;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use structopt::StructOpt;
+
+#[path = "bpf/.output/tracecon.skel.rs"]
+mod tracecon;
+use tracecon::*;
+
+#[derive(Debug, StructOpt)]
+struct Command {
+    /// verbose output
+    #[structopt(long, short)]
+    verbose: bool,
+    /// glibc path
+    #[structopt(long, short, default_value = "/lib/x86_64-linux-gnu/libc.so.6")]
```

Comment:
FWIW: in libbpf-tools (bcc repo) we've suggested parsing `/proc/self/maps` to get path to active libc. But may be out of scope for this example tool.

Reply:
That's interesting, would have to dig into that that though and submit it at a later point. I also noticed in bcc you could attach uprobes using merely a `libc` identifier, I will look into this code.